src/cdc/snapshot_payload.rs

```diff
@@ -1,25 +1,29 @@
+use bon::bon;
+
 use crate::postgres::table_mode::TableMode;
 
 use super::cdc_operator_mode::ModeValueEnum;
 
 #[allow(clippy::too_many_arguments)]
 #[derive(Debug)]
 pub struct CDCOperatorSnapshotPayload {
-    pub bucket_name: String,
-    pub key: String,
-    pub database_name: String,
-    pub schema_name: String,
-    pub included_tables: Vec<String>,
-    pub excluded_tables: Vec<String>,
-    pub mode: ModeValueEnum,
-    pub start_date: Option<String>,
-    pub stop_date: Option<String>,
-    pub source_postgres_url: String,
-    pub target_postgres_url: String,
+    bucket_name: String,
+    key: String,
+    database_name: String,
+    schema_name: String,
+    included_tables: Vec<String>,
+    excluded_tables: Vec<String>,
+    mode: ModeValueEnum,
+    start_date: Option<String>,
+    stop_date: Option<String>,
+    source_postgres_url: String,
+    target_postgres_url: String,
 }
 
+#[bon]
 impl CDCOperatorSnapshotPayload {
     #[allow(clippy::too_many_arguments)]
```

Comment:
You can remove these allows from the functions. `bon` [adds them automatically](https://github.com/elastio/bon/blob/7a4b211741e05adbefa9f9ddc3722e5af3063430/bon-macros/src/builder/builder_gen/input_fn.rs#L245-L256)